app/services/sidekiq_batch_cleanup_service.rb

```diff
@@ -0,0 +1,51 @@
+require 'progress_counter'
+
+class SidekiqBatchCleanupService
```

Comment:
See https://github.com/selleo/pattern#service

```suggestion
class SidekiqBatchCleanupService < Patterns::Service
```

hmm :thinking: , I thought I added it already but seems not. Maybe it was part of managed by operator that then got rewritten